package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "serve-backend": "npx nodemon ./backend/index.js",
```

Comment:
Don't need `serve-backend` script anymore.

Reply:
can it be left there? for development purposes?

Reply:
Of course, I just don't think it is very useful.